#167377529 Deploy application to Heruko

Nennyfills opened the pull request:
#### What does this PR do?
 This pull request deploys vali-1kbdeas application to Heruko 

#### Description of Task to be completed?
- create vali-1kbdeas-staging.herokuapp.com for development
- create vali-1kbdeas.herokuapp.com for production

#### How should this be manually tested?
[vali-1kbdeas-staging.herokuapp.com](https://vali-1kbdeas-staging.herokuapp.com/)
[vali-1kbdeas.herokuapp.com](https://vali-1kbdeas.herokuapp.com)

#### Any background context you want to provide?
The recent heroku documentation states that `Procfile` is no longer required to run a Node.js app on Heroku.  It also stated If no `Procfile` is present in the root directory of the app during the build process, it will check for a start script entry in the package.json file.

#### What are the relevant pivotal tracker stories?
[167377529](https://www.pivotaltracker.com/story/show/167377529)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/45515619/61664877-a1899e00-accb-11e9-9488-93cc104a0a74.png)

#### Questions:
N/A

ibkadeeko reviewed (dismissed):
Looks good to me

successgilli commented on package.json:
```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "heroku-postbuild": "npm run build",
```
Sorry, I may be missing something but I don't see the build script that 'heroku-postbuild' is suppose to run.

iverenshaguy replied:
Also, you can remove the `heroku-postbuild` script because Heroku now runs NodeJS build script by default before pruning the dev dependencies. Please check this article, https://devcenter.heroku.com/changelog-items/1557

iverenshaguy commented:
Good job deploying to Heroku and using the correct branch naming and commit convention.

A few things to note
- Your PR title should be in sentence case i.e #167377529 Deploy application to Heruko.
- Your description should always start with `This pull request ...` not just `Deploy vali....`
- There should be new lines after your commit header and before your commit footer. There should also be a space after `:` in your commit header.

Please make the necessary adjustments

iverenshaguy commented:
Another thing we should do is create a new Heroku account so that we can have our 5 pipelines intact. I'll do that and revert. Please name the new apps `vali-1kbdeas` and `vali-1kbdeas-staging`

iverenshaguy commented:
I cannot find the apps you created on Vali's Heroku dashboard

iverenshaguy commented:
Please fix merge conflicts so that we can merge. Also, add the Procfile needed to run the app on Heroku.

iverenshaguy commented:
Kindly remove the space before the ":" in your commit message i.e `chore(heroku): deploy application to heruko` and update your branch.

iverenshaguy commented:
@Nennyfills Kindly rebase and add a space after the `:` in your header. It should be 
```
chore(heroku): deploy application to heruko
```
 not any of
```
chore(heroku):deploy application to heruko
``` 
or
 ```
chore(heroku) : deploy application to heruko
```
Notice the space after the `:`. This is very important

Nennyfills committed:
chore(heroku): deploy application to heruko

- create vali-ah-staging.herokuapp.com for development
- create vali-ah.herokuapp.com for production
- add postbuild script to the package.json file

[Starts #167377529]

iverenshaguy reviewed (approved):
Good job!
